modules/programs/bash.nix

```diff
@@ -195,6 +204,12 @@ in
         ${cfg.bashrcExtra}
       '';
 
+      home.file.".bash_logout".text = ''
+        # -*- mode: sh -*-
```

Comment:
is that an emacs modeline ? I don't see the point  of adding it here. 
I would rather see no file created if no extra is specified

Reply:
Fixed it